### ISSUE TITLE
Updated Framework crashes my game everytime on startup

### ISSUE THREAD

**Issue by SalmaHayekStan** [praydog/REFramework#867]
**Describe the bug**
Recently updated the framework for RE4R and now my game crashes everytime i start it up. The only way i can play the game is removing the dinput8.dll file from the game's directory (similar to my RE2R issue)

**Upload logs and any crash dumps**
Locate the `re2_framework_log.txt` in your game directory and upload it along with the issue.
If you crashed, upload the `reframework_crash.dmp` file as well.
[re2_framework_log.zip](https://github.com/praydog/REFramework/files/12222306/re2_framework_log.zip)
[reframework_crash.zip](https://github.com/praydog/REFramework/files/12222310/reframework_crash.zip)
github.com/praydog/REFramework/files/12222306/re2_framework_log.zip)

**Identify the REFramework version**
If the REFramework version or commit hash is known, submit this information.
Recent framework release for RE4

**Expected behavior**
Be able to play the game with updated framework

**Desktop and VR (please complete the following information):**
Steam Deck

**Steam Deck/Linux (please complete the following information):**
 - Proton Experimental

**Comment by mochaaaa11** [comment]
you ever get to fix this issue?

**Comment by SalmaHayekStan** [comment]
> you ever get to fix this issue?

Nope I haven’t gotten help with the issue. I guess we have to be super duper specific and precise with the issue lol

**Comment by mochaaaa11** [comment]
yeah i just posted today and he answered.. all i said was my game version was older and it crashes, he told me it doesnt support older versions or pirated copies of the game. idk why he hasnt answered you yet. but if you have older version of the game or pirated copy, that could be the reason for you

**Comment by SalmaHayekStan** [comment]
> yeah i just posted today and he answered lol.. all i said was my game version was older and it crashes, he told me it doesnt support older versions or pirated copies of the game. idk why he hasnt answered you yet. but if you have older version of the game or pirated copy, that could be the reason for you

Hmm now i’m curious because my game copies are legit and I have proof of purchases from steam to back it up and my games are updated regularly so i’m not quite sure what the issue is

**Comment by mochaaaa11** [comment]
i just updated my game and still crashing so idk either. ive seen he wasnt replying for weeks to a lot of open issues on this topic and he answered me today only 20 mins after i posted. maybe the Framework is just outdated or needs a patch and hes working on it.

**Comment by praydog** [comment]
Believe this is an issue with Proton and the current nightly build. I'll need to download Linux or get a Steam Deck which is a barrier to me solving this.

**Comment by mochaaaa11** [comment]
ohhhh i just noticed you are on steam deck. thats a whole new can of worms to open lol. if the issue is happening on an normal windows build then i can imagine the troubleshooting needed to be done on steam deck with experimental build. good luck ahha

**Comment by kristaps777** [comment]
So, would it be safe to say, even if the game itself runs flawlessly on on Linux, the framework might not?
I haven't had any luck launching the game with the framework installed, always crashes on new game/continue.
